source/_components/sensor.fastdotcom.markdown

```diff
@@ -14,58 +14,15 @@ ha_release: 0.26
 ha_iot_class: "Cloud Polling"
 ---
 
-The `fastdotcom` sensor component uses the [Fast.com](https://fast.com/) web service to measure network bandwidth performance.
+The `fastdotcom` sensor uses the [Fast.com](https://fast.com/) web service to measure network bandwidth performance.
 
 <p class='note'>
-Currently fast.com only supports measuring download bandwidth. If you want to measure bandwidth metrics other then download such as ping and upload, utilize the [speedtest](/components/sensor.speedtest) component.
+You must have the [Fast.com component](/components/fastdotcom/) configured to use this sensor.
```

Comment:
Not sure this needs to be a note. Maybe also worth rewording to say that the sensor is created automatically when the component is configured.